workflows/export/vx_export_bmm.go

```diff
@@ -198,7 +199,13 @@ func VXExportToBMM(ctx workflow.Context, params VXExportChildWorkflowParams) (*V
 	}, nil
 }
 
-func makeBMMJSON(ctx workflow.Context, params VXExportChildWorkflowParams, audioResults map[string][]common.AudioResult, normalizedResults map[string]activities.NormalizeAudioResult, chapters []asset.TimedMetadata) ([]byte, error) {
+func makeBMMJSON(
+	ctx workflow.Context,
+	params VXExportChildWorkflowParams,
+	audioResults map[string][]common.AudioResult,
+	normalizedResults map[string]activities.NormalizeAudioResult,
+	chapters []asset.TimedMetadata,
```

Comment:
It might be a bit unfortunate to couple timedmetadata chapters in the bmm export.
I split "GetChapterMeta" up so that it returns the metadata instead of timedmetadata so now it's easier to use that instead.

Reply:
This is a bigger change that is IMO out of scope for this PR.